### PR TITLE
Show scholarship card on right side of scholarship form

### LINKED ISSUE #985: [Bug]: Cannot see description when filling out requirements section in scholarship form
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

I cannot see the description when filling out the requirements section in the scholarship form. This makes it difficult to double check what the requirements should be.

### Steps To Reproduce

1. Go to https://dreamscholars.org/scholarships/new
2. Fill in fields.
3. Click **Next**.
4. Notice you can't double check with the description.

### What devices are you experiencing the problem on?

Desktop, Mobile, Tablet

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge, Other

### LINKED ISSUE #1045: [FR]: Show scholarship card on right side of scholarship form when on bigger screens
### Is there an existing feature request for this?

- [X] I have searched the existing feature requests.

### What is your feature idea?

When I am adding/editing a scholarship I find it frustrating when I have to go back and forward between steps to check the description & fill out the requirements. 

### What alternatives have you considered?

moving the descriptions text area to the `requirements` section

### PR DESCRIPTION
**Pull request by edwingl10**
## Motivation and Context

Fixes #1045. Fixes #985

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] User visible change (users will notice UI or functional changes)

## How Has This Been Tested?

- [x] Existing or new tests cover my changes.
- [ ] Manually tested locally or on the Render PR Server.

## Previewing Changes

1. Click the `onrender.com` URL the **render `bot`** posted below.
2. add / edit a scholarship
4. notice the detail card on the right when viewing through a bigger screen
5. try viewing on mobile

### CODE REVIEW

**Comment by josueetcom** [comment]
I would say this fixes #985

File: src/pages/AddScholarship.jsx [hunk 1]
```diff
@@ -10,51 +10,44 @@ function AddScholarship() {
   const { t } = useTranslation();
 
   return (
-    <Container maxWidth="md" sx={{ p: 2 }}>
```
Comment (josueetcom) [inline src/pages/AddScholarship.jsx:13]: One thing I'm curious about is if we can keep this outer container while maximizing the screen width usage on medium devices. It looks awkward on huge screens (or if you zoom out):

Screen Shot 2022-04-25 at 8 57 02 PM


Would setting `maxWidth` to `lg` or `xl` address this?

File: src/pages/AddScholarship.jsx [hunk 2]
```diff
@@ -10,51 +10,44 @@ function AddScholarship() {
...
-      </Grid>
+      </Container>
 
-      <Paper
-        elevation={2}
-        sx={{
-          zIndex: 1,
-          position: 'relative',
-          p: { xs: 2, sm: 3 },
-          bottom: { md: 40 },
```
Comment (josueetcom) [inline src/pages/AddScholarship.jsx:53]: You might be able to keep this behavior (if you want to) using a Box.

File: src/pages/EditScholarship.jsx [hunk 3]
```diff
@@ -79,12 +78,12 @@ function EditScholarship() {
   }
 
   return (
-    <Container maxWidth="md">
```
Comment (josueetcom) [inline src/pages/EditScholarship.jsx:82]: Similar question as above.

File: src/pages/AddScholarship.jsx (hunk 1 above)
Comment (edwingl10) [reply src/pages/AddScholarship.jsx:13]: maxWdith of 'xl' works.

File: src/pages/AddScholarship.jsx (hunk 2 above)
Comment (edwingl10) [reply src/pages/AddScholarship.jsx:53]: Imma leave it out since the form takes up a longer width than the image, so it might look a lil weird.

**Review by josueetcom — APPROVED** [review]

File: src/components/ScholarshipCard.tsx [hunk 4]
```diff
@@ -109,7 +109,7 @@ export default function ScholarshipCard({
   const CardAreaComponent: React.FC<{
     [key: string]: any;
   }> = detailed ? Box : CardActionArea;
-  const lintIssues = canEdit || preview ? lint(scholarship.data) : [];
+  const lintIssues = style === 'detail' ? lint(scholarship.data) : [];
```
Comment (josueetcom) [inline src/components/ScholarshipCard.tsx:112]: Bro I just noticed this opens up showing lint errors to everyone 😢

**Pull request merged**